packages/orbit-components/src/Itinerary/Itinerary.stories.tsx

```diff
@@ -514,7 +514,12 @@ export const Stop = () => {
 
 export const Detail = () => {
```

Comment:
This is the Detail story, but the props actually exist on the Segment sub-component, so I'd say it makes sense to be on the Segment story. You can still keep this one, bue please also add to the Segment one 🙏

Reply:
Sure, it made sense to me to put it here but you are correct that it makes sense to have it on both.